ui/app/components/project-input-variables/list-item.ts

```diff
@@ -30,7 +30,7 @@ export default class ProjectInputVariablesListComponent extends Component<Variab
     this.variable = variable;
     this.isEditing = isEditing;
     this.isCreating = isCreating;
-    this.storeInitialVariable();
+    this.initialVariable = JSON.parse(JSON.stringify(this.variable));
```

Comment:
What was the motivation for this change?

Reply:
The linter doesn't seem to follow down to `this.storeInitialVariable()` function to see we're setting `initialVariable`, so it thinks we're not setting `initialVariable` in the constructor. I wasn't in love with the solution, happy to hear other suggestions!

Reply:
Ooh that's curious. A limitation of typescript I guess. This solution seems fine.